new-dti-website/components/courses/data/trends_instructors.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Thinking about it more, I also agree with @cchrischen's much earlier comment that we should just maintain NetID's here, and just have `all-members.json` store the source of truth for member data.

Reply:
Maybe we can have this all be in `config.json` to reduce the number of files that need to be edited per semester.

Reply:
This has been on mind too. We have a lot of configs throughout the website right now, and it might be more complicated to edit each of them compared to just editing one big file, since we can just take a pass through the entire file each semester when picking what to update.

Reply:
Deleted `trends_instructors.json` and moved the info to `config.json`.